spec/support/seeds/seeds_e2e.rb

```diff
@@ -358,3 +358,24 @@
     )
   end
 end
+
+##############################################################################
+
+seeder.create_if_doesnt_exist(Podcast, "title", "Test podcast") do
```

Comment:
@aitchiss I did add up the seed data to setup a new podcast and added the worker command too to fetch an episode for that podcast. But while I see a podcast row in the podcasts table; there's no data in the podcast_episodes table. Also, the cypress chrome emulator shows up neither the podcast nor any episode.
![image](https://user-images.githubusercontent.com/47032027/128172978-4e6285d2-8b19-451e-9263-676e1f0550fb.png)

Can you help me figure this out?

Reply:
sorry for the delay - I was on holiday for a while there and just catching up on things! I'll take a look into this and see what I can find 🤔

Reply:
No issues. Let me know if you need any more information regarding this.

Reply:
I opened a [PR](https://github.com/aayush-05/forem/pull/1) against your branch that includes the following

- a static mp3 file (available locally at /media/crow-call.mp3) 
- replacing the `Podcasts::GetEpisodesWorker` call with a PodcastEpisode.create! adding an episode with that file as the media_url

I think the reason that the GetEpisodesWorker wasn't also creating an episode is that we fetch the episodes from the feed in series, then create background jobs to process creation of the episodes from the feed data in parallel, and I don't believe the sidekiq background jobs are being processed when the e2e seed code is run (so the background jobs aren't processed). 

My reason for preferring a locally stored mp3 file to a remote resource for this is that the UI test for playback shouldn't rely on the import episodes flow (we test that on the backend already), and it's better if we don't rely a specific podcast feed location (and episode location) during test time (any number of reasons could come up that the resource becomes inaccessible to us).

Reply:
Thanks for going out of your way and helping @djuber, I couldn't have figured this out without. I too agree on the part that we should have control over test data.

Reply:
Let me know if you need any further help with the cypress test @aayush-05 now there's a test podcast seeded there 🙌  And thanks @djuber for helping us with this - I was stumped too 😄

Reply:
I added the cypress test @aitchiss (can be messy since this is the first time I am writing one); please review

Reply:
Does this require any more changes @aitchiss?